app/src/main/java/com/hannesdorfmann/adapterdelegates4/sample/MainActivity.kt

```diff
@@ -0,0 +1,89 @@
+package com.hannesdorfmann.adapterdelegates4.sample
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.recyclerview.widget.LinearLayoutManager
+import android.content.Intent
+import com.hannesdorfmann.adapterdelegates4.sample.animations.AnimationDiffUtilsActivity
+import com.hannesdorfmann.adapterdelegates4.sample.databinding.ActivityMainBinding
+import com.hannesdorfmann.adapterdelegates4.sample.model.*
+import com.hannesdorfmann.adapterdelegates4.sample.pagination.PaginationActivity
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+
+    private val animals: List<DisplayableItem>
+        get() {
```

Comment:
I think this could be simplified to:

```kotlin
 private val animals: List<DisplayableItem> = mutableListOf( Cat("American Curl"), Cat("Baliness"), ... ).shuffle()
```

Reply:
@sockeqwe  got it. fixed in ec964bba401441d4b2c682d0a835e6a742b3655e